prog/learn_storage.rb

```diff
@@ -12,7 +12,8 @@ def self.parse(s)
   end
 
   label def start
-    q_storage_root = "/var".shellescape.freeze
+    q_storage_root = "/var/storage".shellescape.freeze
+    sshable.cmd("mkdir -p #{q_storage_root}")
```

Comment:
```suggestion
    sshable.cmd("sudo mkdir -p #{q_storage_root}")
```

Reply:
...or should it belong to rhizome? @pykello any opinion given maybe the SPDK daemon will have to have read only access soon, yet data is not particularly sensitive, and multiple unix users seem okay to own the file?

Reply:
Currently it's owned by root. fstab mounts a btrfs partition to there.

Reply:
root probably suffices then, and it's a common choice as well